docs/src/modules/components/TopLayoutCompany.js

```diff
@@ -31,17 +36,29 @@ function TopLayoutCompany(props) {
   const { description, rendered, title } = docs.en;
 
   return (
-    <AppFrame disableDrawer>
+    <BrandingProvider>
+      <AppHeader />
       <Head title={`${title} - MUI`} description={description} />
       <div className={classes.root}>
         <AppContainer component="main" className={classes.container}>
+          <Link
+            href="/careers/#open-roles"
+            rel="nofollow"
+            color="text.secondary"
```

Comment:
I don't get how it works, is this `text.secondary`?

<img width="174" alt="Screenshot 2021-09-20 at 23 58 25" src="https://user-images.githubusercontent.com/3165635/134087374-9779522c-acc1-4707-b5b1-c5cc275994e8.png">

Reply:
Removed `text.secondary`.